gitgud/levels/ramp_up/__init__.py

```diff
@@ -0,0 +1,11 @@
+from collections import OrderedDict
+
+import pkg_resources
+
+from gitgud.levels.util import BasicChallenge
+
+all_challenges = OrderedDict()
+all_challenges['detaching'] = BasicChallenge('detaching', pkg_resources.resource_filename(__name__, '_detaching/'))
+all_challenges['rel refs 1'] = BasicChallenge('rel_refs_1', pkg_resources.resource_filename(__name__, '_rel_refs1/'))
```

Comment:
The command line won't accept names with spaces. I think we should change this name to relrefs1 and relrefs2. Similarly, it is my intent that these folders should be able to be used as python modules, which should have names with only lowercase letters and no underscores. Numbers are fine, and leading underscores are fine. Leading underscores signal that it is only meant to be used by the module and not by people importing the module.